physical/postgresql/postgresql_test.go

```
@@ -444,3 +446,43 @@ func TestPostgreSQLBackend_NoCreateTables(t *testing.T) {
 	logger.Info("Running basic backend tests")
 	physical.ExerciseBackend(t, b)
 }
+
+func TestPostgreSQLBackend_PGEnv(t *testing.T) {
```

interesting how it's requiring comment here, I could swear that it was not mandatory (by linter) for the `test` files, maybe new change?

---

@wslabosz-reply Yeah, it was enforced in the upstream repository. I think what happened was that @bazaah removed the existing precompiled binary in #1292, which was compiled for macOS. 

If you go to an earlier commit, #1251, and look at the code checks, it seems like it didn't actually execute: https://github.com/openbao/openbao/actions/runs/14597860000/job/40948279036

```
Run make ci-vet-codechecker
  make ci-vet-codechecker
  shell: /usr/bin/bash -e {0}
==> Checking that build is using go version >= 1.24.2...
==> Using go version 1.24.2...
go: not generating in packages in dependency modules
go: not generating in packages in dependency modules
```

It seems to silently fail to execute on Linux, if you drop in a MacOS binary. Joy!

---

@wslabosz-reply I fixed this before merging. Thanks!